lispy.el

```diff
@@ -7375,9 +7375,18 @@ Defaults to `error'."
       (forward-line 1))
     (move-marker end nil)))
 
+(defvar lispy-no-indent-modes '(minibuffer-inactive-mode
+                                comint-mode)
+  "List of major modes where `indent-for-tab-command' should not be used.
+`lispy--indent-for-tab' will do nothing if the current mode or any of its parent
+modes is in this list.")
+
+(declare-function parent-mode-list "parent-mode")
 (defun lispy--indent-for-tab ()
   "Call `indent-for-tab-command'."
-  (unless (or (memq major-mode '(minibuffer-inactive-mode comint-mode))
+  (require 'parent-mode)
+  (unless (or (cl-some (lambda (mode) (memq mode lispy-no-indent-modes))
+                       (parent-mode-list major-mode))
```

Comment:
I'd never heard of `parent-mode-list` - what's the benefit of using it here rather than something like 
```emacs-lisp
(apply derived-mode-p lispy-no-indent-modes)
```
?

Reply:
None in this case. Thanks, I'll update the pull request.

Reply:
Good catch with the `#'` - was still in scheme mode last night, heh.